src/network/message_timer_system.cpp

```diff
@@ -0,0 +1,131 @@
+#include "platform/i_platform.h"
+#include "message_timer_system.h"
+#include "../core/program_state.h"
+
+namespace network {
+
+MessageTimerSystem::MessageTimerSystem()
+    : mScene( Scene::Get() )
+{
+}
+
+
+void MessageTimerSystem::Init()
+{
+    using boost::filesystem::path;
+    Filesys& FSys = Filesys::Get();
+    PathVect_t Paths;
+    FSys.GetFileNames( Paths, "actors" );
+    for (auto const& Path : Paths)
+    {
+        if (Path.extension().string() != ".network")
```

Comment:
this should be put into some fs_utils ns, we do the same thing all over the code